.ci/jenkins/base_image.dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM ubuntu:16.04
+
+ARG http_proxy
+ARG https_proxy
+ENV http_proxy=${http_proxy} https_proxy=${https_proxy}
+
+# nGraph dependencies
+RUN apt-get -y update && apt-get -y install git build-essential cmake clang-3.9 git curl zlib1g zlib1g-dev libtinfo-dev \
```

Comment:
Add at the beginning 
```
apt update --fix-missing &&
```
and at the end  apt autoremove -y && apt clean all

Reply:
done